test/puppetlabs/puppetdb/query/event_counts_test.clj

```diff
@@ -57,12 +48,12 @@
           (doseq [[order expected] [[:ascending  [count2 count1]]
                                     [:descending [count1 count2]]]]
             (testing order
-              (let [actual (:result (raw-event-counts-query-result
-                                     version
-                                     ["=" "certname" "foo.local"]
-                                     "containing_class"
-                                     {}
-                                     {:order_by [[:successes order]]}))]
+              (let [actual (query-event-counts
+                            version
+                            ["=" "certname" "foo.local"]
+                            "containing_class"
+                            {}
+                            {:order_by [[:successes order]]})]
                 (is (= actual expected)))))))
```

Comment:
Yeah that's definitely worth it's own ticket, its a problem throughout all our tests